src/mean.js

```diff
@@ -0,0 +1,21 @@
+var _curry1 = require('./internal/_curry1');
+var sum = require('./sum');
+
+
+/**
+ * Returns the mean of the given list of numbers.
+ *
+ * @func
+ * @memberOf R
+ * @category Math
+ * @sig [Number] -> Number
+ * @param {Array} xs
+ * @return {Number}
+ * @example
+ *
+ *      R.mean([2, 7, 9]); //=> 6
+ *      R.mean([]); //=> NaN
+ */
+module.exports = _curry1(function mean(xs) {
```

Comment:
Usually we'd use the name `list`.  Did you want `xs` to distinguish a list of numeric values